Week5/app/src/main/java/com/isabelledionisius/mymovieapp/DetailActivity.kt

```diff
@@ -0,0 +1,36 @@
+package com.isabelledionisius.mymovieapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import kotlin.properties.Delegates
+
+class DetailActivity : AppCompatActivity() {
+    /**
+     * properties to be displayed in the detail view
+     */
+    lateinit var movies:Movie
```

Comment:
Don't use lateinit vars for views. Use either ViewBindings or Kotlin Android Extensions.

Reply:
I cleaned up my code by getting rid of findViewById and replaced them with Kotlin Extensions. Worked fine for the views. However, I wonder how I can use an Kotlin Android Extension for "lateinit var movies:Movie" since movies is not a view but an instance of Movie. Does that work with extensions?